src/pupil_labs/rec_export/export.py

```diff
@@ -536,19 +536,54 @@ def _process_fixations(
         use_cached_optic_flow=True,
         progress=progress,
     )
-    events = pd.DataFrame(events).query("label == 'fixation'").reset_index(drop=True)
-    events.drop(columns=["label"], inplace=True)
+    events = pd.DataFrame(events)
+
+    # Common fields
     events.rename(
         columns={"start_t": "start timestamp [ns]", "end_t": "end timestamp [ns]"},
         inplace=True,
     )
+    events["duration [ms]"] = (
+        events["end timestamp [ns]"] - events["start timestamp [ns]"]
+    ) // 1_000_000
```

Comment:
I know this PR isn't the origin of the issue but I'd round this so that a 4999999ns saccade ends up as 5ms instead of 4ms, eg.
```python
round((end_ns - start_ns) / 1e6)
```

This is also how it's handled in cloud.